tests/workflows/integration_tests/execution/test_workflow_with_detection_renaming.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
```

Comment:
great tests coverage

Reply:
@PawelPeczek-Roboflow Hey Pawel, I couldn't figure out or find a good example of passing Input Parameters into UQL operations for tests. Instead I created a hacky work around with parameterized tests to replace the Workflow Specification to test scenarios. Feel free to change this if you'd like; as I would also like to learn how to pass input parameters into UQL operations for future work.